package.json

```
@@ -7,7 +7,7 @@
     "start": "nodemon --exec babel-node src/index.js",
     "build": "babel ./src -d ./dist -s --copy-files",
     "lint": "eslint --ignore-path .gitignore .",
-    "test": "npm run lint && ava",
+    "test": "npm run lint && node ./test/api.test.js",
```

running directly with `node` probably won't work, should use Babel:

```suggestion
    "test": "npm run lint && babel-node ./test/api.test.js",
```